routing/geometry.cpp

```diff
@@ -51,14 +51,18 @@ namespace routing
 {
 // RoadGeometry ------------------------------------------------------------------------------------
 RoadGeometry::RoadGeometry(bool oneWay, double speed, Points const & points)
-  : m_points(points), m_speed(speed), m_isRoad(true), m_isOneWay(oneWay)
+  : m_points(points), m_speed(speed), m_isOneWay(oneWay)
```

Comment:
Could you please reorder fields or parameters to the ctor, as there is one-to-one correspondence?